<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>collectorWidgetClass</class>
 <widget class="QWidget" name="collectorWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>collectorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="entities"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="offersOfItem"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>bid</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
